common/djangoapps/student/migrations/0049_auto__add_languageproficiency__add_unique_languageproficiency_code_use.py

```diff
@@ -173,7 +173,7 @@ def backwards(self, orm):
             'courseware': ('django.db.models.fields.CharField', [], {'default': "'course.xml'", 'max_length': '255', 'blank': 'True'}),
             'gender': ('django.db.models.fields.CharField', [], {'db_index': 'True', 'max_length': '6', 'null': 'True', 'blank': 'True'}),
             'goals': ('django.db.models.fields.TextField', [], {'null': 'True', 'blank': 'True'}),
-            'has_profile_image': ('django.db.models.fields.BooleanField', [], {'default': 'False'}),
+            'profile_image_version': ('django.db.models.fields.CharField', [], {'max_length': '16', 'null': 'True'}),
```

Comment:
i think the physical files are a good source of truth for that bit of
metadata as well, no ?

On Tue, Mar 31, 2015 at 5:23 PM, Max Rothman <notifications@github.com>
wrote:

> In
> common/djangoapps/student/migrations/0049_auto__add_languageproficiency__add_unique_languageproficiency_code_use.py
> <https://github.com/edx/edx-platform/pull/7517#discussion_r27524609>:
>
> > @@ -173,7 +173,7 @@ def backwards(self, orm):
> >              'courseware': ('django.db.models.fields.CharField', [], {'default': "'course.xml'", 'max_length': '255', 'blank': 'True'}),
> >              'gender': ('django.db.models.fields.CharField', [], {'db_index': 'True', 'max_length': '6', 'null': 'True', 'blank': 'True'}),
> >              'goals': ('django.db.models.fields.TextField', [], {'null': 'True', 'blank': 'True'}),
> > -            'has_profile_image': ('django.db.models.fields.BooleanField', [], {'default': 'False'}),
> > +            'profile_image_version': ('django.db.models.fields.CharField', [], {'max_length': '16', 'null': 'True'}),
>
> +1 to not using this field to mean "last modified". The source of truth in
> that case is S3. However, I can see a rationale for using an int field for
> data size, but I'd just a hash algo that outputs an int.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/7517/files#r27524609>.
>